puffin_http/Cargo.toml

```diff
@@ -14,7 +14,10 @@ include = ["**/*.rs", "Cargo.toml", "README.md"]
 
 [dependencies]
 anyhow = "1.0"
-crossbeam-channel = "0.5"
+async-executor = "1.4.1"
```

Comment:
Including an async run time into the library by default is something that we would like to prevent. Especially our users, among our selfs, might be using different runtimes already making them statically depended on an other one.

Reply:
Maybe a new crate can be created that has this specific runtime which then executes the puffin_http

Reply:
Migrate to async involves the use of async, await keyword in multiple place.
I don't think it's possible to create a new crate that use puffin_http as base and add "async" on top.

I think the best option to keep both, is use `feature` management of crates with some code duplication.
Are you interested in having me tried this option ?

PS: I haven't tested, but think than directly use of async-executor, instead of async-std task keep runtime local to crate, and avoid conflict if application use other async runtime.

Reply:
And so long as possible do not include any runtimes. Doesn't seem necessary, or do you think otherwise?

Reply:
The interest to use async-executor directly is to keep all the puffin server work in a dedicated thread, "isolated" of the rest of the app. So I think having a runtimes (async-executor) included is important. And as it's related to a LocalExecutor only use in puffin_http/server.rs / in puffin-server thread, I supposed it's could work with app than use other runtime (like Tokio) without interference.
Possibly it can be deactivated with option for user than don't want an additional thread and prefer than puffin_server work run on the same executor than the rest of the app.
I think it better to target by default the user who don't have async and just want add profiling simply and working "out of the box". 
PS: I'm not sure than puffin profiler is adapted for full async app. So not sure this is the type of user to target with this.